controllers/component_build_controller_pac.go

```diff
@@ -1014,20 +1016,24 @@ func (r *ComponentBuildReconciler) UnconfigureRepositoryForPaC(ctx context.Conte
 			}
 		}
 
+		action_done = "delete"
 		prUrl, err = gitClient.UndoPaCMergeRequest(repoUrl, mrData)
-		return baseBranch, prUrl, "delete", err
```

Comment:
Why did you change the logic here and return later in the function?

Reply:
explained in call